apis/accesscontextmanager/v1beta1/zz_generated.resolvers.go

```diff
@@ -42,7 +44,7 @@ func (mg *AccessLevelCondition) ResolveReferences( // ResolveReferences of this
 	if err != nil {
 		return errors.Wrap(err, "mg.Spec.ForProvider.AccessLevel")
 	}
-	mg.Spec.ForProvider.AccessLevel = reference.ToPtrValue(rsp.ResolvedValue)
+	mg.Spec.ForProvider.AccessLevel = ptr.To(rsp.ResolvedValue)
```

Comment:
Since the function was removed in crossplane-runtime [here](https://github.com/crossplane/crossplane-runtime/commit/3cdde89278abd217d394e7b146660ef8720e76bb), the fallback to `ptr` would result in different behaviour. `ToPtrValue` would return nil for an empty string whereas `ptr.To` [would not](https://pkg.go.dev/k8s.io/utils/ptr#To).

Reply:
@sergenyalcin my understanding is that this is a problem because `spec.initProvider` is meant to be write-once. if empty references are leaking in when nothing was specified at creation, it might explain the confusing error seen in the uptest failure: https://github.com/crossplane-contrib/provider-upjet-gcp/actions/runs/14729137527/job/41338930150.

Reply:
xref https://github.com/crossplane/crossplane-tools/pull/106